manifests/gcp_marketplace/schema.yaml

```diff
@@ -77,13 +77,21 @@ x-google-marketplace:
       properties:
         images.metadatawriter:
           type: FULL
+    cacheserver:
+      properties:
+        images.cacheserver:
+          type: FULL
+    cachedeployer:
+      properties:
+        images.cachedeployer:
+          type: FULL
   deployerServiceAccount:
     roles:
       - type: ClusterRole        # This is a cluster-wide ClusterRole
         rulesType: CUSTOM        # We specify our own custom RBAC roles
         rules:
-          - apiGroups: ['apiextensions.k8s.io']
-            resources: ['customresourcedefinitions']
+          - apiGroups: ['apiextensions.k8s.io', 'rbac.authorization.k8s.io']
+            resources: ['customresourcedefinitions', 'clusterroles', 'clusterrolebindings']
```

Comment:
why add these? Is it because you add additional role? I'm not pretty sure whether it can work.

Here is a guide for your local test before submit codes.

https://github.com/kubeflow/pipelines/pull/3378/files README.md line 21

Build: 
gcloud builds submit --config=.cloudbuild.yaml --substitutions=COMMIT_SHA="$(git rev-parse HEAD)" --project=ml-pipeline-test

Install:
mpdev install  --deployer=gcr.io/ml-pipeline-test/hosted/$(git rev-parse HEAD)/deployer:$MM_VER \
    --parameters='{"name": "'$APP_INSTANCE_NAME'", "namespace": "'$NAMESPACE'"}'

Reply:
Adding this is because I add additional cluster scope role. I did the local test, without this line the code doesn't work.